openfisca_aotearoa/variables/income_tax/working_for_families/working_for_families.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Titled_Property, Person, Family
+from numpy import clip, floor
+
+class received_income_tested_benefit_as_per_social_security(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    label = u'Number of Persons classified as receiving an income tested benefit'
```

Comment:
It seems strange to me that a “Number of Persons” can be expressed as a Boolean value and would be associated with a Person 🤔 Are these labels accurate (repeated below)? 🙂